cob_msgs/msg/PowerState.msg

```diff
@@ -1,10 +1,9 @@
-# This message communicates the state of the power system.
 std_msgs/Header header
-float64 voltage                     # [V]
```

Comment:
Why did you remove the useful comments on utilized units?

Reply:
I'm not sure why, but they caused issues with the ros2 message generation.  Unfortunately the output is not helpfull either. Somewhere when tokenizing/building a Tree out of the .msg Files a hickup happens. I tested the PowerState message in multiple ways (complete manual rewrite to not have stray control characters, ...) but as soon as I added `# [V]` or some other comments (`# [Ah]` was also crashing) the error occured and cannot be removed until the comment was removed again. 

I would loved to just adapt the CMakeList.txt and the package.xml but unfortunately this took some time and was strange. 

Any ROS2 Guru knowing whats wrong here?

Error log:
```
/home/jannik/git/ros2_nav_ws/build/cob_msgs/rosidl_adapter/cob_msgs/msg/PowerState.idl
Error processing idl file: /home/jannik/git/ros2_nav_ws/build/cob_msgs/rosidl_adapter/cob_msgs/msg/PowerState.idl
 /home/jannik/git/ros2_nav_ws/build/cob_msgs/rosidl_adapter/cob_msgs/msg/PowerState.idl
Error processing idl file: /home/jannik/git/ros2_nav_ws/build/cob_msgs/rosidl_adapter/cob_msgs/msg/PowerState.idl
Traceback (most recent call last):
  File "/opt/ros/dashing/lib/rosidl_generator_dds_idl/rosidl_generator_dds_idl", line 41, in <module>
    sys.exit(main())
  File "/opt/ros/dashing/lib/rosidl_generator_dds_idl/rosidl_generator_dds_idl", line 36, in main
    args.additional_service_templates,
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_generator_dds_idl/__init__.py", line 51, in generate_dds_idl
    generate_files(generator_arguments_file, mapping, additional_context, keep_case=True)
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_cmake/__init__.py", line 90, in generate_files
    raise(e)
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_cmake/__init__.py", line 70, in generate_files
Traceback (most recent call last):
    idl_file = parse_idl_file(locator)
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_parser/parser.py", line 64, in parse_idl_file
  File "/opt/ros/dashing/lib/rosidl_generator_dds_idl/rosidl_generator_dds_idl", line 41, in <module>
    content = parse_idl_string(string, png_file=png_file)
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_parser/parser.py", line 73, in parse_idl_string
    sys.exit(main())
  File "/opt/ros/dashing/lib/rosidl_generator_dds_idl/rosidl_generator_dds_idl", line 36, in main
    content = extract_content_from_ast(tree)
    args.additional_service_templates,
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_parser/parser.py", line 144, in extract_content_from_ast
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_generator_dds_idl/__init__.py", line 51, in generate_dds_idl
    generate_files(generator_arguments_file, mapping, additional_context, keep_case=True)
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_cmake/__init__.py", line 90, in generate_files
    add_message_members(msg, struct_defs[0])
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_parser/parser.py", line 367, in add_message_members
    raise(e)
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_cmake/__init__.py", line 70, in generate_files
    idl_file = parse_idl_file(locator)
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_parser/parser.py", line 64, in parse_idl_file
    annotations = get_annotations(member)
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_parser/parser.py", line 512, in get_annotations
    content = parse_idl_string(string, png_file=png_file)
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_parser/parser.py", line 73, in parse_idl_string
    content = extract_content_from_ast(tree)
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_parser/parser.py", line 144, in extract_content_from_ast
    get_const_expr_value(const_expr)
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_parser/parser.py", line 561, in get_const_expr_value
    add_message_members(msg, struct_defs[0])
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_parser/parser.py", line 367, in add_message_members
    annotations = get_annotations(member)
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_parser/parser.py", line 512, in get_annotations
    return get_string_literal_value(child, allow_unicode=False)
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_parser/parser.py", line 592, in get_string_literal_value
    get_const_expr_value(const_expr)
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_parser/parser.py", line 561, in get_const_expr_value
    assert len(string_literal.children) == 1
AssertionError
    return get_string_literal_value(child, allow_unicode=False)
  File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_parser/parser.py", line 592, in get_string_literal_value
    assert len(string_literal.children) == 1
AssertionError

```

Reply:
Might be a bug in `rosidl_parser`.